<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Quality Guide</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>
Select a project to start Data Quality.
</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Calibration</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="4" column="1">
          <widget class="QComboBox" name="comboFromDB"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBias"/>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboFlatField"/>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineEditBias"/>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="lineEditFlatField"/>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="lineEditFromDB"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboDark"/>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="lineEditDark"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Note</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Status</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="Bias">
           <property name="text">
            <string>Bias</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="Dark">
           <property name="text">
            <string>Dark</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QToolButton" name="FlatField">
           <property name="text">
            <string>Flat Field</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="fromDB">
           <property name="text">
            <string>From DB</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Data</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4"/>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Query</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="queryDB"/>
         </item>
         <item>
          <widget class="QPushButton" name="runQueryDB">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Filter</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Report</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTextEdit" name="textEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
